frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -728,5 +735,24 @@ export const buildEChartsSeries = (
     );
   }
 
+  if (chartModel.otherSeriesModel) {
+    // TODO handle area chart
+    series.push(
+      buildEChartsBarSeries(
+        chartModel.transformedDataset,
+        chartModel.xAxisModel,
+        chartModel.yAxisScaleTransforms,
+        chartMeasurements,
+        chartModel.otherSeriesModel,
+        settings,
+        0, // TODO fix after updating y-axis split
```

Comment:
Assigning the other series and handling y-axis auto split will be done in a later PR